pkg/scheduler/framework/plugins/noderesources/fit.go

```diff
@@ -311,7 +311,11 @@ func fitsRequest(podRequest *preFilterState, nodeInfo *framework.NodeInfo, ignor
 				continue
 			}
 		}
-		if rQuant > (nodeInfo.Allocatable.ScalarResources[rName] - nodeInfo.Requested.ScalarResources[rName]) {
+
+		// Skip in case request quantity is zero
+		if rQuant == 0 {
```

Comment:
let's add this to the head of the loop, it's more efficient.

Reply:
done